From 75065513634795934563f1671503a9c63114cec0 Mon Sep 17 00:00:00 2001
From: Jacob Stiffler <j-stiffler@ti.com>
Date: Wed, 16 Sep 2015 10:35:59 -0400
Subject: [PATCH 5/5] HACK: remoteproc: Prevent backtrace for RSC_INTMEM
 deprecation warning

* The backtrace during boot makes it appear that there is a serious
  problem even though it is a warning about a feature being deprecated
  in the near future.
* Change the WARN() to a dev_warn()

Signed-off-by: Jacob Stiffler <j-stiffler@ti.com>
---
 drivers/remoteproc/remoteproc_core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/remoteproc/remoteproc_core.c b/drivers/remoteproc/remoteproc_core.c
index aa3e325..8e6ca80 100644
--- a/drivers/remoteproc/remoteproc_core.c
+++ b/drivers/remoteproc/remoteproc_core.c
@@ -870,7 +870,7 @@ static int rproc_handle_intmem(struct rproc *rproc, struct fw_rsc_intmem *rsc,
 	void *va;
 	int ret;
 
-	WARN(1, "RSC_INTMEM is deprecated. Please do not use this resource type to support loading into internal memories.\n");
+	dev_warn(dev, "RSC_INTMEM is deprecated. Please do not use this resource type to support loading into internal memories.\n");
 
 	if (sizeof(*rsc) > avail) {
 		dev_err(dev, "intmem rsc is truncated\n");
-- 
2.6.4

